src/main/docs/guide/httpServer/customBinding.adoc

```diff
@@ -0,0 +1,43 @@
+
+== Argument binding
```

Comment:
Even though the argument binding API is generic, this section should only be framed around the HTTP server and request binders.